modules/auxiliary/scanner/http/title.rb

```diff
@@ -0,0 +1,97 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'cgi'
+
+class Metasploit3 < Msf::Auxiliary
+  # Exploit mixins should be called first
+  include Msf::Exploit::Remote::HttpClient
+  # Scanner mixin should be near last
+  include Msf::Auxiliary::Scanner
+
+  def initialize
+    super(
+      'Name'        => 'HTTP HTML <title> tag content grabber',
+      'Description' => 'Generates a GET request to the webservers provided and returns the server header, HTML title attribute and location header (if set). Useful for rapidly identifying all webservers on a network and identifying interesting hosts en mass.',
+      'Author'       => ['Stuart Morgan <stuart.morgan[at]mwrinfosecurity.com>'],
+      'License'     => MSF_LICENSE,
+    )
+
+    register_options(
+      [
+        OptBool.new('STORE_NOTES', [ true, 'Store the captured information in notes. Use "notes -t http.title" to view', true ]),
+        OptBool.new('SHOW_ERRORS', [ true, 'Show error messages relating to grabbing titles on the console', true ]),
+        OptBool.new('SHOW_TITLES', [ true, 'Show the titles on the console as they are grabbed', true ])
+      ], self.class)
+
+    deregister_options('VHOST')
+  end
+
+  def run
+    if datastore['STORE_NOTES'] == false && datastore['SHOW_ERRORS'] == false && datastore['SHOW_TITLES'] == false
+      print_error("Notes storage is false, errors have been turned off and titles are not being shown on the console. There isn't much point in running this module.")
+    else
+      super
+    end
+  end
+
+  def run_host(target_host)
+    begin
+        # Send a normal GET request
+        res = send_request_cgi('uri'          => '/',
+                               'method'       => 'GET')
+
```

Comment:
To keep it more readable, the code should be written like:
```ruby
res = send_request_cgi(
  'uri'    => '/',
  'method' => 'GET'
)
```
Not a big deal. Just make it more readable